lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -290,11 +278,14 @@ class Object < ObjectAttribute
     end
 
     class Operation < PolicyElement
-      has_and_belongs_to_many :policy_element_associations, class_name: 'PolicyMachineStorageAdapter::ActiveRecord::PolicyElementAssociation', join_table: 'operations_policy_element_associations'
```

Comment:
```
def operation_sets
  Assignment.ascendants_of(self).where(type: PolicyMachineStorageAdapter::ActiveRecord::OperationSet.to_s)
```
think this would be useful? It looks like you want it lower in the file.